bazel/compiler_id.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Given an argument that is the path to a compiler, tries to determine an
+# identification string for the compiler according to the CMake-defined IDs:
+# https://cmake.org/cmake/help/v3.5/variable/CMAKE_LANG_COMPILER_ID.html
+
+if [[ ! -e "${1}" ]]; then
```

Comment:
What does existence have to do with usage?

Reply:
>> Just since we're about to execute it... ?? Let me know if you want me to change it to something else.
> 
> On second though, you're right. Changed to a more normal $# -ne 1 check. Fixed. thanks.

Alright!